core/coreobjects/include/coreobjects/core_event_args_ids.h

```diff
@@ -41,6 +41,22 @@ namespace core_event_ids
     constexpr Int TagsChanged = 110;
 }
 
+enum class CoreEventId
```

Comment:
Feels a little hacky that OpenDAQ events are defined in coreobjects

Reply:
Technically we could place all of it into opendaq, but then the property object implementation would not be able to create its own arguments.

As a side-note: I've added the enum class for the event IDs because I wanted to use a switch statement. Should we use the `constexpr` above, the enum class, or both?